docs/yaml_structure.md

```diff
@@ -1,2 +0,0 @@
-
```

Comment:
what this file intended for? if nothing, please remove

Reply:
It is removed